website/content/api-docs/health.mdx

```diff
@@ -197,9 +197,10 @@ the following selectors and filter operations being supported:
 | `ServiceTags` | In, Not In, Is Empty, Is Not Empty                 |
 | `Status`      | Equal, Not Equal, In, Not In, Matches, Not Matches |
 
-## List Nodes for Service
+((list-nodes-for-service))
```

Comment:
hmm, this did not work. I'm trying to find a way to keep the link text the same while changing the readable text.  I'll try something else.

Reply:
```suggestion
## List Nodes for Service ((#list-nodes-for-service))
```
This format should do it.